app/helpers/SessionAnswersHelper.scala

```diff
@@ -473,4 +475,20 @@ class SessionAnswersHelper @Inject()(uploadJourneyRepository: UploadJourneyRepos
     userRequest.answers.getAnswer[String](SessionKeys.doYouWantToAppealBothPenalties).contains("no") &&
       userRequest.answers.getAnswer[LocalDate](SessionKeys.dateCommunicationSent).exists(_.isBefore(dateTimeNow.minusDays(appConfig.daysRequiredForLateAppeal)))
   }
+
+  def viewAppealRows()(implicit userRequest: UserRequest[_], messages: Messages): Seq[QuestionAnswersRow] = {
```

Comment:
I think viewAppealRows could be renamed slightly, this func name sounds like it will show something whereas it's retrieving something:
```suggestion
  def getSubmittedAnswers()(implicit userRequest: UserRequest[_], messages: Messages): Seq[QuestionAnswersRow] = {
```